tests/test_cli_validation.py

```
@@ -0,0 +1,178 @@
+"""
+Test CLI argument validation and both entry points.
+
+This module tests the CLI interfaces to ensure proper argument validation
+and that both entry points (mad-spark and mad_spark_alt) work correctly.
+"""
+
+import subprocess
+import sys
+import pytest
+from unittest.mock import patch, Mock
```

Ruff (0.12.2)

`unittest.mock.patch` imported but unused

Remove unused import

(F401)